fuse/dir.go

```diff
@@ -38,7 +39,7 @@ func (d *dir) children(ctx context.Context) (*plugin.EntryMap, error) {
 		return plugin.ListWithAnalytics(ctx, updatedEntry.(plugin.Parent))
 	}
 
-	return nil, fuse.ENOENT
+	return nil, syscall.ENOENT
```

Comment:
Why this change?

Reply:
https://github.com/bazil/fuse/commit/e2f0bde20aef2c6f9a3da6d1d2d3cf2abf4ebe2b

Reply:
Actually that deprecated all `fuse.E*`. I'll update those and mention it in the commit message.